homeassistant/components/mobile_app/webhook.py

```diff
@@ -240,6 +242,32 @@ async def webhook_fire_event(hass, config_entry, data):
     return empty_okay_response()
 
 
+@WEBHOOK_COMMANDS.register("stream_camera")
+@validate_schema({vol.Required(ATTR_CAMERA_ENTITY_ID): cv.string})
+async def webhook_stream_camera(hass, config_entry, data):
+    """Handle a request to HLS-stream a camera."""
+    camera = hass.states.get(data[ATTR_CAMERA_ENTITY_ID])
+
+    if camera is None:
+        return webhook_response(
+            {"success": False}, registration=config_entry.data, status=HTTP_BAD_REQUEST,
+        )
+
+    resp = {"mjpeg_path": "/api/camera_proxy_stream/%s" % (camera.entity_id)}
```

Comment:
this is defined in `CameraImageView` but i'm not sure if there's a proper way to get it out of the place it's registered?